feat(backend-api): include vbzrx amounts for TVL

AoDev opened the pull request:
Here is my attempt at including vbzrx in our calculation of the TVL.

What I am unsure of, is if I needed to also update the total supply in the way I did it. 

https://app.clubhouse.io/bzx1/story/532/update-tvl-calculation-for-the-explorer

Kharabet commented on packages/bzx-api/src/fulcrum/index.js:
```diff
@@ -685,6 +691,18 @@ export default class Fulcrum {
+  /**
+   * @param {string} addressErc20
+   */
+  async getErc20TotalSupply(addressErc20) {
```
Well, actually this is not the supply that we count in stats. To get the one which we need, you have to initiate iToken's contract and call its `totalAssetSupply` function. But, as we have no iToken contract for vBZRX  (ivBZRX token) we couldn't get the desired supply metric.

Kharabet commented on packages/bzx-api/src/fulcrum/index.js:
```diff
@@ -554,6 +554,12 @@ export default class Fulcrum {
       swapRates[iTokens.map((x) => x.name).indexOf('bzrx')]
     ).dividedBy(10 ** 18)
     const monthlyReward = new BigNumber(10300000).times(bzrxUsdPrice)
+    const [vbzrxLockedAmount, vbzrxTotalSupply] = await Promise.all([
```
So, we don't need `vbzrxTotalSupply` at all. `vbzrxLockedAmount` should be considered in BZRX's TVL and $TVL

AoDev commented:
@Kharabet I've updated the PR. The TVL related change is in this commit: 
https://github.com/bZxNetwork/fulcrum_ui/pull/341/commits/6167d8faa5bc1deaa59f37c89c5f71f3e39502ed

The other commits are where I added eslint and just ran a quick eslint fix all on the fulcrum index file.
